app/services/analytics_events.rb

```diff
@@ -2431,14 +2401,76 @@ def idv_in_person_usps_request_enroll_exception(
     )
   end
 
-  # Tracks individual enrollments that succeed during GetUspsProofingResultsJob
-  # @param [String] reason why did this enrollment pass?
+  # GetUspsProofingResultsJob is beginning. Includes some metadata about what the job will do
+  # @param [Integer] enrollments_count number of enrollments eligible for status check
+  # @param [Integer] reprocess_delay_minutes minimum delay since last status check
+  def idv_in_person_usps_proofing_results_job_started(enrollments_count:, reprocess_delay_minutes:,
+                                                      **extra)
+    track_event(
+      'GetUspsProofingResultsJob: Job started',
+      enrollments_count: enrollments_count,
+      reprocess_delay_minutes: reprocess_delay_minutes,
+      **extra,
+    )
+  end
+
+  # GetUspsProofingResultsJob has completed. Includes counts of various outcomes encountered
+  # @param [Integer] enrollments_checked number of enrollments eligible for status check
+  # @param [Integer] enrollments_errored number of enrollments for which we encountered an error
+  # @param [Integer] enrollments_expired number of enrollments which expired
+  # @param [Integer] enrollments_failed number of enrollments which failed identity proofing
+  # @param [Integer] enrollments_in_progress number of enrollments which did not have any change
+  # @param [Integer] enrollments_passed number of enrollments which passed identity proofing
+  def idv_in_person_usps_proofing_results_job_completed(enrollments_checked:, enrollments_errored:,
+                                                        enrollments_expired:, enrollments_failed:,
+                                                        enrollments_in_progress:,
+                                                        enrollments_passed:, **extra)
```

Comment:
Soft preference to follow the convention of other methods in this file with multi-line keyword arguments (one-per-line, indented one)

```suggestion
  def idv_in_person_usps_proofing_results_job_completed(
    enrollments_checked:,
    enrollments_errored:,
    enrollments_expired:,
    enrollments_failed:,
    enrollments_in_progress:,
    enrollments_passed:,
    **extra
  )
```

Reply:
Fixed in 082bbed130d50238eb30aa4ef31f48625b4576c5